### PR TITLE
Feature/performance blocks produced

### PR DESCRIPTION
**Pull request by jak-pan**

### CODE REVIEW

**Comment by jak-pan** [comment]
This requires client to call validator={accountId} query to get blocksProduced upon clicking on detail in UI

File: packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import { FullScreenModal } from 'ui'
+import { useBooleanState } from 'hooks'
+import { NOOP } from 'utils'
+
+storiesOf('UI|FullScreenModal', module)
+  .add('permanent', () => {
+    const textKnob = text(
+      'Text',
+      'Cupcake ipsum dolor sit amet bonbon pudding sugar plum jujubes. Biscuit sweet fruitcake tart cupcake cotton candy jelly beans gummi bears tart. Cheesecake gummies sweet roll candy lollipop jelly. Chocolate bar powder muffin liquorice croissant halvah wafer sugar plum. Jujubes brownie lollipop soufflé. Sweet ice cream cookie jelly beans sweet roll fruitcake cupcake sweet. Chocolate bar gingerbread dessert. Cheesecake pudding danish halvah jujubes toffee. Cheesecake cotton candy jujubes candy canes pastry toffee marshmallow. Danish toffee gingerbread sweet roll sweet roll. Icing macaroon oat cake. Sugar plum donut sesame snaps halvah muffin wafer candy jelly. Bonbon danish donut marshmallow cheesecake marzipan biscuit powder.',
```
Comment (jak-pan) [inline packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx]: This should not be in our code.

File: packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    )
+
+    return <FullScreenModal onClose={NOOP}>{textKnob}</FullScreenModal>
+  })
+  .add('with animation', () => {
+    const [modalVisible, showModal, hideModal] = useBooleanState()
+
+    const textKnob = text(
+      'Text',
+      'Cupcake ipsum dolor sit amet bonbon pudding sugar plum jujubes. Biscuit sweet fruitcake tart cupcake cotton candy jelly beans gummi bears tart. Cheesecake gummies sweet roll candy lollipop jelly. Chocolate bar powder muffin liquorice croissant halvah wafer sugar plum. Jujubes brownie lollipop soufflé. Sweet ice cream cookie jelly beans sweet roll fruitcake cupcake sweet. Chocolate bar gingerbread dessert. Cheesecake pudding danish halvah jujubes toffee. Cheesecake cotton candy jujubes candy canes pastry toffee marshmallow. Danish toffee gingerbread sweet roll sweet roll. Icing macaroon oat cake. Sugar plum donut sesame snaps halvah muffin wafer candy jelly. Bonbon danish donut marshmallow cheesecake marzipan biscuit powder.',
```
Comment (jak-pan) [inline packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx]: Same here

File: packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx (hunk 1 above)
Comment (Andkoo) [reply packages/client/stories/ui/FullScreenModal/FullScreenModal.stories.tsx]: It's only for story, not for actual component, the text is visible only in storybook.

**Pull request merged**